File: Python/secao06/Exercicio07.py
# input
n1 = int(input("Informe um número 1/4: "))
n2 = int(input("Informe um número 2/4: "))
n3 = int(input("Informe um número 3/4: "))
n4 = int(input("Informe um número 4/4: "))

# processing
q1 = n1 * n1
q2 = n2 * n2
q3 = n3 * n3
q4 = n4 * n4

if q3 >= 100:
    print("Num3: {0}, Quadrado: {1}".format(n3, q3))
else:
    print("Num1: {0}, Quadrado: {1}".format(n1, q1))
    print("Num2: {0}, Quadrado: {1}".format(n2, q2))
    print("Num3: {0}, Quadrado: {1}".format(n3, q3))
    print("Num4: {0}, Quadrado: {1}".format(n4, q4))

File: C/Exercices/secao03/exercicio07.c
/*
Desenvolva um algoritmo que:
a) Leia 4 (quatro) números;
b) Calcule o quadrado de cada um;
c) se o valor resultante do quadrado do terceiro for >= 1000, imprima-o e finalize;
d) Caso contrário, imprima os valores lidos e seus respectivos quadrados.
*/

#include <stdio.h>

int main()
{
    //variáveis
    int num1, num2, num3, num4, q1, q2, q3, q4;

    //entradas
    printf("Informe o número 1: ");
    scanf("%d", &num1);
    printf("Informe o número 2: ");
    scanf("%d", &num2);
    printf("Informe o número 3: ");
    scanf("%d", &num3);
    printf("Informe o número 4: ");
    scanf("%d", &num4);

    //processamento
    q1 = num1 * num1;
    q2 = num2 * num2;
    q3 = num3 * num3;
    q4 = num4 * num4;

    if (q3 >= 1000)
    {
        printf("%d", q3);
    }
    else
    {
        printf("Número 1: %d. Quadrado: %d\n", num1, q1);
        printf("Número 2: %d. Quadrado: %d\n", num2, q2);
        printf("Número 3: %d. Quadrado: %d\n", num3, q3);
        printf("Número 5: %d. Quadrado: %d", num4, q4);
    }
    
}
